backend/migrations/20220822091356_introduce_endpoints.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE endpoint
+(
```

Comment:
You will require a permissioned_as which is distinct from created_by and create bearer token belonging to permissioned_as for any request that reach this (and meet criterion)

Reply:
Reminder that permissioned_as values are  either:
`u/<user>`
or `g/<group>`